cli/app.go

```diff
@@ -7,6 +7,7 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
[nit] Would you mind using the standard library `errors` instead of `github.com/pkg/errors`?